nvtabular/workflow.py

```diff
@@ -41,11 +41,11 @@ class BaseWorkflow:
 
     Parameters
     -----------
-    cat_names : list of str
```

Comment:
We shouldn't make this change - 'list of str' is correct here. See the numpy docstring guide
https://numpydoc.readthedocs.io/en/latest/format.html#sections (search for 'list of str').